internal/controller/instaslice_controller.go

```diff
@@ -387,7 +448,7 @@ func (*InstasliceReconciler) extractProfileName(limits v1.ResourceList) string {
 			if len(match) > 1 {
 				profileName = match[1]
 			} else {
-				log.Log.Info("No match found")
+				fmt.Printf("No match found")
```

Comment:
Why to go away from logging with levels?

Reply:
Thanks, this logging now is not needed, will remove

Reply:
Done.